pkgs/build-support/rust/fetch-cargo-tarball/default.nix

```diff
@@ -76,6 +76,17 @@ in stdenv.mkDerivation ({
     # Override the `http.cainfo` option usually specified in `.cargo/config`.
     export CARGO_HTTP_CAINFO=${cacert}/etc/ssl/certs/ca-bundle.crt
 
+    if grep --pcre2 '^source = "(?!registry)' Cargo.lock; then
```

Comment:
```
grep: unrecognized option '--pcre2'
```

Reply:
perhaps we can just do `^source = "git+` since we don't really know how to deal with non-git deps anyway?

Reply:
That was my `alias grep=rg` in my environment 😅

Reply:
Fixed